docs/analyze.md

```diff
@@ -192,6 +192,8 @@ This document introduces Event Log DataCollector. We will start with a brief ove
 
 #### Introduction
 Event Log DataCollector is a Windows only DataCollector that is used to get event logs logged into Windows Event Viewer during test execution. Event logs are saved in a file `Event Log.xml` and this file is available as Attachment as part of test result report (trx).
+When enabled, Event Log DataCollector generates one `Event Log.xml` file for entire test session. `Event Log.xml` files are also generated corresponding to all test cases as well, to provide a granular view of events logged while executing a test case.
```

Comment:
I don't feel good about eventlog.xml for each test case though. We should probably add a switch, maybe ?